Nie-ontologies/Generic-ontologies/literature.ttl

```diff
@@ -138,22 +138,22 @@ literature:DescriptiveLiterature
 literature:Expression
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/literature#>;
 	a rdfs:Class;
-	rdfs:label "literary expression"@en, "literarische Expression"@de, "expression littéraire"@fr, "espressione letteraria"@it;
+	rdfs:label "literary expression"@en, "literarischer Ausdruck"@de, "expression littéraire"@fr, "espressione letteraria"@it;
```

Comment:
'Ausdruck' has to be changed back to 'Expression', also in other ontologies, cf.
https://www.ifla.org/files/assets/cataloguing/frbr/frbr_2009_de.pdf:
Vorwort zur deutschen Übersetzung (on the 5th page, before the content table): paragraph 3.